### DISCUSSION TITLE
How to run Seleniumbase case for parallel testing?

### DISCUSSION THREAD

**Discussion by GitSleep8** [seleniumbase/SeleniumBase#2574] [Q&A] [answered]
Hi Michael, 

Seleniumbase has the "-n" option to run test cases much faster, but is there another option to run test case for parallel testing, just like pytest-parallel?
For example, I have only one test case, but I need to open 10 browser windows at the same time to run this test case in parallel.
Is there a way for it?

Thanks
Shawn

**Comment by mdmintz** [comment, marked as answer]
Have you seen [examples/test_repeat_tests.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_repeat_tests.py)?

The examples there use `pytest -n NUM` for the multiple processes via `pytest-xdist`.

You can also spin up multiple drivers in the same test. See [examples/test_multiple_drivers.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_multiple_drivers.py).

**Reply by GitSleep8** [reply]
Thanks a lot, it's working to set the repeat number first and then run `pytest -n NUM`
